nvfuser/CMakeLists.txt

```diff
@@ -0,0 +1,323 @@
+if(NOT BUILD_NVFUSER)
+  return()
+else()
+  cmake_minimum_required(VERSION 3.0)
```

Comment:
Have you tested it on 3.0? Perhaps we should use 3.18 as head PyTorch?

Reply:
Good point. I'll bump this up to 3.18 as with PyTorch then.